asa/ja-jp/content/topics/adv_server_access/docs/ad-client-configuration-advanced.htm

```diff
@@ -294,20 +294,20 @@ <h2>はじめに</h2>
                                                         <li>この高度なクライアント構成が行われる各サーバーに<span class="mc-variable workflows_connectors.Advanced_Server_Access variable">Advanced Server Access</span>のクライアントがインストールされている必要があります。</li>
                                                     </ul>
                                                     <h2>Forward Agent（エージェントを転送する）</h2>
-                                                    <p><span class="mc-variable workflows_connectors.Advanced_Server_Access variable">Advanced Server Access</span>により最初に接続したサーバーにローカルSSHエージェントを転送するには、この機能を利用します。</p>
+                                                    <p><span class="mc-variable workflows_connectors.Advanced_Server_Access variable">Advanced Server Access</span>を介して最初に接続したサーバーにローカルSSHエージェントを転送するには、この機能を利用します。この機能が機能するには、接続先のサーバーでTCPフォワーディングが許可されている必要があります。<span class="filepath">/etc/ssh/sshd_config</span>ファイル内のステートメントがTCPフォーワーディングを禁止しているときは（<span class="code">AllowTcpForwarding no</span>など）、これを<span class="code">AllowTcpForwarding yes</span>に変更た上でSSHDを再起動します。</p>
                                                     <div class="noteOkta">
                                                         <p class="noteContent">この機能はWindowsクライアントと互換性がありません。</p>
                                                     </div>
                                                     <ol>
                                                         <li value="1"><span class="mc-variable workflows_connectors.Advanced_Server_Access variable">Advanced Server Access</span>のダッシュボードで<span class="uicontrol">［Projects（プロジェクト）］</span>に移動します。</li>
-                                                        <li value="2"><span class="wintitle">［Details（詳細）］</span>タブで<span class="uicontrol">［Client Configuration（クライアント構成）］</span>に移動します。</li>
+                                                        <li value="2"><span class="wintitle">［詳細］</span>タブで<span class="uicontrol">［Client Configuration（クライアント構成）］</span>に移動します。</li>
```

Comment:
@pengyao-okta regression? or perhaps we need to add `wintitle` class?
(here and below)